pkg/kubectl/rolling_updater_test.go

```diff
@@ -692,6 +693,45 @@ func TestUpdate_progressTimeout(t *testing.T) {
 	}
 }
 
+func TestUpdate_canRollbackFromAFailedRcToAReadyRc(t *testing.T) {
```

Comment:
I'll take a look. I assume it can't be because the test above it, `TestUpdate_progressTimeout`, is also a timeout scenario and not part of `TestUpdate`.